crates/matrix-sdk-crypto/src/store/integration_tests.rs

```diff
@@ -568,29 +568,22 @@ macro_rules! cryptostore_integration_tests {
             }
 
             #[async_test]
-            async fn test_load_inbound_group_sessions_by_room_id() {
-                let dir = "load_inbound_group_session_by_room_id";
+            async fn test_get_inbound_group_sessions_by_room_id() {
```

Comment:
please could you add another test that checks we get an empty list back when there are no sessions in the room?